gnocchi/rest/api.py

```diff
@@ -1380,6 +1404,10 @@ def _search(self, **kwargs):
             details=details,
             history=history,
             **pagination_opts)
+
+        for r in resources:
+            r.set_json_attrs(json_attrs)
```

Comment:
So I put my functional hat and I don't like that. :(

Can we just `return [r.jsonify(json_attrs) for resources]` instead?